apps/sptribs/sptribs-cron-clear-inactive-dss-draft-cases/sptribs-cron-clear-inactive-dss-draft-cases.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: sptribs-cron-clear-inactive-dss-draft-cases
+spec:
+  releaseName: sptribs-cron-clear-inactive-dss-draft-cases
+  values:
+    job:
+      image: hmctspublic.azurecr.io/sptribs/case-api:prod-9545153-20240807154219 #{"$imagepolicy": "flux-system:sptribs-case-api"}
+      keyVaults:
+        sptribs:
+          secrets:
+            - name: app-insights-connection-string
+              alias: app-insights-connection-string
+            - name: uk-gov-notify-api-key
+              alias: UK_GOV_NOTIFY_API_KEY
+            - name: s2s-case-api-secret
+              alias: S2S_SECRET
+            - name: idam-secret
+              alias: IDAM_CLIENT_SECRET
+            - name: idam-systemupdate-username
+              alias: IDAM_SYSTEM_UPDATE_USERNAME
+            - name: idam-systemupdate-password
+              alias: IDAM_SYSTEM_UPDATE_PASSWORD
+      environment:
+        TASK_NAME: SystemClearInactiveDssDraftCasesTask
+      schedule: 0 8 * * 1-5
```

Comment:
Can we schedule this for some other time? Global search migration has been configured for the same time, I believe.